central/signatureintegration/datastore/validate.go

```diff
@@ -91,6 +94,41 @@ func validateCosignCertificateVerification(configs []*storage.CosignCertificateV
 		if _, err := cryptoutils.UnmarshalCertificatesFromPEM([]byte(config.GetCertificatePemEnc())); err != nil {
 			multiErr = multierror.Append(multiErr, errors.Wrap(err, "unmarshalling certificate PEM"))
 		}
+
+		if ctlog := config.GetCertificateTransparencyLog(); ctlog.GetEnabled() {
+			if ctlogPubKey := ctlog.GetPublicKeyPemEnc(); ctlogPubKey != "" {
```

Comment:
nitpick: maybe we can consider flattening nested ifs:
```suggestion
		ctlog := config.GetCertificateTransparencyLog()
		ctlogPubKey := ctlog.GetPublicKeyPemEnc()
		if !ctlog.GetEnabled() || ctlogPubKey == "" {
			continue
		}
```